gno.land/pkg/gnoclient/integration_test.go

```diff
@@ -113,6 +114,70 @@ func TestCallMultiple_Integration(t *testing.T) {
 	assert.Equal(t, expected, got)
 }
 
+func TestCallMultipleEvent_Integration(t *testing.T) {
```

Comment:
No, I'm sorry. This is not testing gnoclient, it is testing the chain. You should write a `txtar` test where you run gnokey with a transaction generated by combining two transactions, ie. running `sign` and `broadcast` from an already generated transaction result.

This may sound nitpicky, but honestly I already hate that we have txtar tests because we don't have a system to properly unit-tests things on the TM2 side. We will not test chain behaviour with the gnoclient package. If something fails here, it should be gnoclient's fault, not the chain code (other code should scream at you when that happens).

Reply:
AFAIK, gnokey in txtar doesn't support sign(not sure about broadcast).
Is there way around to test it?